suite/tests/CMakeLists.txt

```diff
@@ -287,6 +287,24 @@ string(REGEX REPLACE "/lib([36])" "/ext/lib\\1"
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/bin")
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/bin")
+
+function (set_per_config_ouput_to_match_single_config)
```

Comment:
This function is already present in the top-level CMakeLists.txt which means it is available for use here.  A duplicate copy is not necessary.  What problem are you trying to solve?